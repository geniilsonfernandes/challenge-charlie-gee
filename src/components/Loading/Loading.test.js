import renderWithTheme from "../../utils/test/renderWithTheme";
import { screen } from "@testing-library/react";
import { Loading } from ".";

describe("<Loading/.>", () => {
  it("should render loading component", () => {
    const { container } = renderWithTheme(<Loading />);
    expect(screen.getByLabelText(/loading/i)).toBeInTheDocument();
    expect(container).toMatchSnapshot();
  });
});
