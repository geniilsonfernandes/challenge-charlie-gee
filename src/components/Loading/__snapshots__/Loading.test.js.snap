// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loading/.> should render loading component 1`] = `
<div>
  <div
    aria-label="loading"
    class="sc-gsnTZi ivoJFd"
  >
    <span
      class="sc-bczRLJ fJjrnV"
      data-icon="("
      title="not found"
    />
  </div>
</div>
`;
